### PR TITLE
[MRG] Adding Appveyor CI

### PR DESCRIPTION
**Pull request by vighneshbirodkar**
Addresses #14

### CODE REVIEW

**Comment by vighneshbirodkar** [comment]
@fabianp @mblondel @MechCoder Can you have a look ?

File: ci_scripts/appveyor/install.ps1 [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        $webclient.DownloadFile($GET_PIP_URL, $GET_PIP_PATH)
+        Write-Host "Executing:" $python_path $GET_PIP_PATH
+        & $python_path $GET_PIP_PATH
+    } else {
+        Write-Host "pip already installed."
+    }
+}
+
+
+function DownloadMiniconda ($python_version, $platform_suffix) {
```
Comment (fabianp) [inline ci_scripts/appveyor/install.ps1]: I don't think this is used plus appveyor now has miniconda installed by default

**Comment by fabianp** [comment]
Hey. Thanks for working on this. I believe it is correct but the appveyor install script might be unnecessarily complicated. Now appveyor has miniconda pre-installed so there is no need to install the Python official version. Using miniconda has the added benefit that it is possible to build conda packages (this is how we build lightning conda win packages)

**Comment by vighneshbirodkar** [comment]
@fabianp Have a look now. I am not yet building conda recipes. I prefer to leave that to a different PR

**Comment by fabianp** [comment]
Awesome, +1 from me

**Comment by vighneshbirodkar** [comment]
@MechCoder Please have a look.

**Comment by vighneshbirodkar** [comment]
@mblondel Could you review this ? It already has one +1

**Comment by MechCoder** [comment]
@lesteve might be the right person to ping for issues like these.

**Comment by lesteve** [comment]
> @lesteve might be the right person to ping for issues like these.

From a quick look at it, this looks good to me. Full disclosure I have never written something like this from scratch but rather always looked at what scikit-learn was doing and adapted it for other projects. I am guessing this is what you have done as well.

**Comment by vighneshbirodkar** [comment]
@fabianp Can you merge this ? @lesteve also gave it an OK

**Comment by fabianp** [comment]
Great, merging

**Pull request merged**